testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4WD/build_mesh/config_base_mesh.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<config case="base_mesh">
+	<get_file dest_path="bathymetry_database" file_name="SRTM15_plus_earth_relief_15s.nc">
+		<mirror protocol="wget" url="https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-ocean/bathymetry_database/"/>
+	</get_file>
+
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS" dest="jigsaw_to_MPAS"/>
+	<add_link source_path="script_test_dir" source="." dest="define_base_mesh"/>
```

Comment:
@pwolfram, I think the `define_base_mesh.py` file needs to be added

Reply:
Thanks-- do you see it now post force-push?

Reply:
Verified it exists now-- thanks

Reply:
Got it! Thanks.